app/controllers/elevator_bookings_controller.rb

```diff
@@ -52,28 +67,19 @@ def prep_bookings
     @elevator_bookings = ElevatorBooking.all.map do |b|
       {
         id: b.id, endTime: b.end, startTime: b.start, unit: b.unit,
-        ownerType: b.ownerType, name1: b.name1, name2: b.name2,
+        name1: b.name1, name2: b.name2, user: b.user,
         phoneDay: b.phone_day, phoneNight: b.phone_night, deposit: b.deposit,
-        moveType: b.moveType, approved: b.approved
+        moveType: b.moveType, approved: b.approved, moveIn: b.in, moveOut: b.out
       }
     end
   end
 
   # Only allow a list of trusted parameters through.
   def elevator_booking_params
     params.require(:elevator_booking).permit(
-      :user_id,
-      :start,
-      :end,
-      :unit,
-      :ownerType,
-      :name1,
-      :name2,
-      :phone_day,
-      :phone_night,
-      :deposit,
-      :moveType,
-      :approved
+      :user_id, :start, :end, :unit, :ownerType, :name1, :name2,
```

Comment:
If ownerType is to be removed, this should be taken out too.

Reply:
Done!